components/infobox/extensions/commons/role_of.lua

```diff
@@ -41,7 +41,7 @@ function RoleOf.get(args)
 		name = teamData.id,
 		link = teamData.link,
 		flag = teamData.nationality,
-	}})
+	} --[[@as standardOpponent]] })
```

Comment:
This one might indicate a deeper issue

Reply:
it is only casting away nil
could also rewrite it to
```lua
local opponent = Opponent.readOpponentArgs{....}
---@cast -nil

return tostring(OpponentDisplay.InlineOpponent{opponent = opponent})
```
it can not be nil due to the setup (and nil check on `teamData` some lines above it)